lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -23,10 +23,9 @@
     web,
 )
 from galaxy.exceptions import ObjectInvalid
-from galaxy.managers import (
-    api_keys,
-    users,
-)
+from galaxy.managers import api_keys
+from galaxy.managers import base as managers_base
+from galaxy.managers import users
```

Comment:
I never noticed that isort was forcing the `from ... import ... as ...` lines to be separate from the `from ... import ...` ones.
Should we add `combine_as_imports=true` to `.isort.cfg` ? This would make isort change the lines above to:

```python
from galaxy.managers import (
    api_keys,
    base as managers_base,
    users,
)
```

Clearly I would open this as a separate PR, just looking for feedback. Such change would affect 46 files.

Reply:
I had the imports lines you have there and isort undid it - so I would prefer the suggested change I think. It is both more natural and looks more consistent to me.

Reply:
Cool, I'll open a PR!